site/docs/hosting/vps.md

```diff
@@ -0,0 +1,121 @@
+# VPS
+
+A virtual private server, mostly known as VPS is a virtual machine running in the cloud with its users having the full control of its filesystem.
+There are tons of advantages in using a VPS over things like Heroku and microservices to run your bots.
```

Comment:
This is only partially true. Yes, you have more control, but it also means that you have more responsibility. Heroku manages system updates and security patches automatically without downtime or intervention, which is very hard to do manually. I am uncomfortable with painting a black-and-white picture here.